scope3_methodology/api/api.py

```diff
@@ -89,6 +92,21 @@ async def startup_event():
     property_defaults[PropertyChannel.STREAMING] = Property.load_default_yaml(
         "generic", property_defaults_file_path, PropertyChannel.STREAMING.value
     )
+    property_defaults[PropertyChannel.STREAMING] = Property.load_default_yaml(
```

Comment:
this line is duplicated above?

Reply:
this just means the code works twice as well.

Reply:
You guys are hysterical.